### PR TITLE
Fix some doc weirdness

### PR DESCRIPTION
**Pull request by IainNZ**

### CODE REVIEW

**Comment by IainNZ** [comment]
Also `bfs_tree` and `dfs_tree` take a second argument, but thats not what the docs say. Not sure whats happening there?

**Comment by sbromberger** [comment]
Oh, yeah, they require a source vertex. Thanks.

File: doc/pathing.md [hunk 1]
```diff
@@ -3,10 +3,10 @@ various utility functions. Where appropriate, edge distances may be passed in as
 matrix of real number values. The matrix should be indexed by `[src, dst]` (see [Getting Started](gettingstarted.html) for more information).
 
 ### Graph Traversal
-`Graph traversal` refers to a process that traverses vertices of a graph following certain order (starting from user-input sources). This package implements three traversal schemes:
-` `BreadthFirst`,
-` `DepthFirst`, and
-` `MaximumAdjacency`.
+"Graph traversal" refers to a process that traverses vertices of a graph following certain order (starting from user-input sources). This package implements three traversal schemes:
```
Comment (sbromberger) [inline doc/pathing.md]: Ugh. This was global search and replace. Could you change this to *Graph traversal*? Terms are emphasized.

**Comment by IainNZ** [comment]
Updated. I figure it was some search and replace madness :D

**Comment by sbromberger** [comment]
Thank you!

**Pull request merged**